ZombieSharp/EventModule.cs

```diff
@@ -76,7 +76,11 @@ private HookResult OnPlayerConnectFull(EventPlayerConnectFull @event, GameEventI
 
             Logger.LogInformation($"Player: {client.PlayerName} is fully connected with {client.Slot}");
```

Comment:
Use structured logs. Do not use string interpolation for logging purposes. You are losing some additional information and some performance. Please, check it out https://www.youtube.com/watch?v=XV7N21Dwa-U